templates/newrelic-stack-set.yml

```diff
@@ -16,6 +16,10 @@ Parameters:
     is created using the name you specify. This custom policy includes only minimal permissions 
     that allow New Relic to monitor your Lambda functions. Note that you are responsible for 
     managing a custom policy.'
+  NewRelicTrustedPrincipal:
```

Comment:
Is this the value we are passing from the main launch template?

Reply:
this value is not passed from main launch template at the moment, default value is used during launch

Reply:
If we are not passing it from the main template than I don't see the value of add it as parameter. Idea behind parameterizing it was to allow users to explicitly provide a 3rd party account id which they are giving permission to take actions in their AWS Account.

Reply:
Agreed and I've document it here https://github.com/aws-quickstart/quickstart-ct-newrelic-one/issues/27

this will require another PR to modify the documentation, and perhaps we will also cover different partition, i.e. govcloud if needed.

I added this param on `newrelic-stack-set.yml` as stop gap from hard-coded principal in the template.

Reply:
When do you think we will have the PR to address it?